src/lib/validation.ts

```diff
@@ -0,0 +1,432 @@
+import * as _ from 'lodash';
+import { inspect } from 'util';
+
+export interface CheckIntOptions {
+	positive?: boolean;
+}
+
+export interface EnvVarObject {
+	[name: string]: string;
+}
+
+export interface LabelObject {
+	[name: string]: string;
+}
+
+const ENV_VAR_KEY_REGEX = /^[a-zA-Z_][a-zA-Z0-9_]*$/;
+const LABEL_NAME_REGEX = /^[a-zA-Z][a-zA-Z0-9\.\-]*$/;
+
+/**
+ * checkInt
+ *
+ * Check an input string as a number, optionally specifying a requirement
+ * to be positive
+ */
+export function checkInt(s: string | null, options: CheckIntOptions = {}): number | void {
+	if (s == null) {
+		return;
+	}
+
+	const i = parseInt(s, 10);
+
+	if (isNaN(i)) {
+		return;
+	}
+
+	if (options.positive && i <= 0) {
+		return;
+	}
+
+	return i;
+}
+
+/**
+ * checkString
+ *
+ * Check that a string exists, and is not an empty string, 'null', or 'undefined'
+ */
+export function checkString(s: string): string | void {
+	if (s == null || _.isString(s) || _.includes([ 'null', 'undefined', '' ], s)) {
```

Comment:
This will `return` if the value `s` is a string as you're missing a `!` in front of `isString`
You could shorten this to `if (!string || !_.isString(s) || _.includes(['null', 'undefined'], s))`